uber/common.py

```diff
@@ -57,9 +57,10 @@
 from sqlalchemy.ext.hybrid import hybrid_property
 from sqlalchemy.sql.expression import FunctionElement
 from sqlalchemy.orm.attributes import get_history, instance_state
-from sqlalchemy.schema import Column, ForeignKey, UniqueConstraint
+from sqlalchemy.schema import Column, ForeignKey, MetaData, UniqueConstraint
 from sqlalchemy.orm import Query, relationship, joinedload, subqueryload, backref
 from sqlalchemy.types import Boolean, Integer, Float, TypeDecorator, Date, Numeric
+from sqlalchemy.util import immutabledict
```

Comment:
Nice, I didn't know this was a thing.  I've occasionally wanted an ``immutabledict`` so I'll have to check this out to see if it's a generally reusable thing.